src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationPostConfigureOptions.cs

```diff
@@ -59,19 +59,19 @@ public class SuperOfficeAuthenticationPostConfigureOptions : IPostConfigureOptio
                         options.MetadataAddress += ".well-known/openid-configuration";
                     }
 
-                    if (options.RequireHttpsMetadata && !options.MetadataAddress.StartsWith("https://", StringComparison.OrdinalIgnoreCase))
+                    if (!options.MetadataAddress.StartsWith("https://", StringComparison.OrdinalIgnoreCase))
                     {
                         throw new InvalidOperationException("The MetadataAddress or Authority must use HTTPS unless disabled for development by setting RequireHttpsMetadata=false.");
                     }
 
                     options.ConfigurationManager = new ConfigurationManager<OpenIdConnectConfiguration>(
                         options.MetadataAddress,
                         new OpenIdConnectConfigurationRetriever(),
-                        new HttpDocumentRetriever(options.Backchannel) { RequireHttps = options.RequireHttpsMetadata })
-                    {
-                        RefreshInterval = options.RefreshInterval,
-                        AutomaticRefreshInterval = options.AutomaticRefreshInterval,
-                    };
+                        new HttpDocumentRetriever(options.Backchannel))
+                        {
+                            RefreshInterval = TimeSpan.FromSeconds(30),
+                            AutomaticRefreshInterval = TimeSpan.FromDays(1)
```

Comment:
Nit: sorting

```suggestion
                            AutomaticRefreshInterval = TimeSpan.FromDays(1),
                            RefreshInterval = TimeSpan.FromSeconds(30),
```

Reply:
Change the sorting, and dependency on Options.